cvat-core/src/annotations.ts

```diff
@@ -137,6 +137,7 @@ export async function exportDataset(
     instance,
     format: string,
     saveImages: boolean,
+    allImages: boolean,
```

Comment:
It's better to call it `includeOnlyAnnotated` or `onlyAnnotated`, to reflect what the option does comparing to the "normal" behavior.